tests/regression_tests/deplete_with_transport/test.py

```diff
@@ -122,8 +122,18 @@ def test_full(run_in_tmpdir, problem, multiproc):
     n_tallies = np.empty(N + 1, dtype=int)
 
     # Get statepoint files for all BOS points and EOL
+    runtimes = {}
     for n in range(N + 1):
         statepoint = openmc.StatePoint(f"openmc_simulation_n{n}.h5")
+        runtime = statepoint.runtime
+        if n == 0:
+            for measure, time in runtime.items():
+                runtimes.update({measure: np.array([time])})
+        else:
+            for measure, time in runtime.items():
+                current = runtimes[measure]
+                updated = np.append(current, time)
+                runtimes.update({measure: updated})
```

Comment:
At the top of the module, if you add `from collections import defaultdict`, you can then do:
```suggestion
    runtimes = defaultdict(list)
    for n in range(N + 1):
        statepoint = openmc.StatePoint(f"openmc_simulation_n{n}.h5")
        for measure, time in statepoint.runtime.items():
            runtimes[measure].append(time)
```